server/src/test/java/org/candlepin/service/impl/DefaultEntitlementCertServiceAdapterTest.java

```diff
@@ -853,8 +853,9 @@ protected DefaultEntitlementCertServiceAdapter initCertServiceAdapter() {
     public void ensureV3CertificateCreationOkWhenConsumerSupportsV3Dot1Certs()
         throws Exception {
 
-        consumer.setFact("system.certificate_version", "3.3");
+        consumer.setFact("system.certificate_version", "3.4");
```

Comment:
Certificate version is a good candidate for extraction. It is used at a lot of places so it will make a bit it easier to update the version in the future.

Reply:
Done.